libs/blocks/chart/utils.js

```diff
@@ -34,20 +34,12 @@ export function formatExcelDate(date) {
   let newDate;
 
   if (!Number.isNaN(+date)) {
-    const hours = Math.floor((+date % 1) * 24);
-    const minutes = Math.floor((((+date % 1) * 24) - hours) * 60);
-    const offsetUTC = 24 - (new Date().getTimezoneOffset() / 60);
-
-    newDate = new Date(Date.UTC(0, 0, +date, hours - offsetUTC, minutes));
+    newDate = +date > 99999
+      ? new Date(+date * 1000)
+      : new Date(Math.round((+date - (1 + 25567 + 1)) * 86400 * 1000));
```

Comment:
wouldn't mind some var names for these numbers, if it makes sense to do so.

Reply:
This is the best I found for an explanation https://gist.github.com/christopherscott/2782634